pkg/alertmanager/multitenant_test.go

```diff
@@ -2163,6 +2167,25 @@ receivers:
 	require.Contains(t, err.Error(), errRateLimited.Error())
 }
 
+func TestMultitenantAlertmanager_computeFallbackConfig(t *testing.T) {
+	// If no fallback config is set, it should return a valid empty config.
```

Comment:
```suggestion
	// If no fallback configration is set, it returns a valid empty configuration.
```

Reply:
To remove ambiguity and raise the tone of confidence, reword instances of "should" throughout to what will or will not happen.